sdk/open_api/api/market_data_api.py

```
@@ -2535,3 +2536,520 @@ def _get_prices_serialize(
+    @validate_call
+    async def get_spot_market_summary(
+        self,
+        symbol: Annotated[str, Field(strict=True, description="Trading symbol (e.g., BTCRUSDPERP)")],
+        _request_timeout: Union[
+            None,
+            Annotated[StrictFloat, Field(gt=0)],
+            Tuple[
+                Annotated[StrictFloat, Field(gt=0)],
+                Annotated[StrictFloat, Field(gt=0)]
+            ]
+        ] = None,
```

Ruff (0.15.12)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)